core/cat/looking_glass/cheshire_cat.py

```diff
@@ -125,29 +126,94 @@ def load_language_model(self) -> BaseLanguageModel:
         -----
         Bootstrapping is the process of loading the plugins, the natural language objects (e.g. the LLM), the memories,
         the *Main Agent*, the *Rabbit Hole* and the *White Rabbit*.
-
         """
-
+        
         selected_llm = crud.get_setting_by_name(name="llm_selected")
+        self._llm_modalities = {"image_uri": False, "image_url": False, "audio": False}
+
+        def _load_test_image():
+            """Return the base64 data for the test image."""
+            with open("cat/loading_cat.jpg", "rb") as image_file:
```

Comment:
where is that file?

Reply:
Perhaps rather than uploading a file, it is better to use the one I download to test image url support